src/lib/blueprint.ts

```diff
@@ -304,8 +312,42 @@ const createRequest = (
     methods: [uppercaseMethod],
     semanticMethod: uppercaseMethod,
     preferredMethod: uppercaseMethod,
-    parameters: createParameters(operation),
+    // @ts-expect-error TODO: resolve property and request properties difference between required and isRequired
+    parameters: createRequestBody(operation),
+  }
+}
+
+const createRequestBody = (
+  operation: OpenapiOperation,
+): RequestBodyProperty[] => {
+  if (!('requestBody' in operation) || operation.requestBody === undefined) {
+    return []
   }
+
+  const requestBody = operation.requestBody
+
+  if (
+    requestBody.content?.['application/json']?.schema?.properties === undefined
+  )
+    return []
+
+  const schema = requestBody.content['application/json'].schema
+
+  if (schema.type !== 'object' || schema.properties == null) {
+    return []
+  }
+
+  const requiredProperties = schema.required ?? []
+
+  return Object.entries(schema.properties).map(
+    ([name, property]: [string, any]): RequestBodyProperty => ({
+      name,
+      type: property.type ?? 'string',
+      format: property.format,
+      description: property.description ?? '',
+      isRequired: requiredProperties,
```

Comment:
```suggestion
      isRequired: requiredProperties.includes(name),
```

Reply:
I think I got confused and thought isRequired could be an array of required parameters 😓